src/AutorestSwift/View Models/ResponseViewModel.swift

```diff
@@ -33,14 +33,17 @@ enum ResponseBodyType: String {
     case body
     /// Service returns no response data, only a status code
     case noBody
+    case stringBody
+    case intBody
+    case jsonBody
```

Comment:
Since these are now in ExceptionResponseBodyType, do they still need to be here?